node/container.go

```diff
@@ -597,6 +596,25 @@ func (a *HostAgent) setupContainer(tenantID string, svc *service.Service, instan
 	return ctr, state, nil
 }
 
+func (a *HostAgent) getParentName(svc *service.Service) string {
+	logger := plog.WithFields(log.Fields{
+		"servicename": svc.Name,
+		"serviceid":   svc.ID,
+	})
+
+	parentID := svc.ParentServiceID
+	if parentID == "" {
+		return ""
+	}
+	parent, err := a.getService(parentID)
+	if err != nil {
+		logger.WithError(err).Debug("Unable to retrieve service")
```

Comment:
this should read "unable to read parent service" and should reference the parentID, otherwise it looks like we can't read the child service.